src/region_watcher.rs

```diff
@@ -8,41 +8,55 @@ const REGION_BACKOFF_RETRIES: u32 = 10;
 const REGION_BACKOFF_MIN_WAIT: Duration = Duration::from_secs(5);
 const REGION_BACKOFF_MAX_WAIT: Duration = Duration::from_secs(3600); // 60 minutes
 
-pub type MessageSender = watch::Sender<Option<RegionParams>>;
-pub type MessageReceiver = watch::Receiver<Option<RegionParams>>;
+pub type MessageSender = watch::Sender<RegionParams>;
+pub type MessageReceiver = watch::Receiver<RegionParams>;
+
+pub fn current<T>(receiver: &watch::Receiver<T>) -> T
```

Comment:
`current` is such an overloaded word. A more descriptive name or a comment would help here.

Reply:
Renamed to `current_value`